build_tools/cmake/riscv.toolchain.cmake

```diff
@@ -23,14 +23,15 @@ set(RISCV_HOST_TAG linux)
 set(RISCV_TOOL_PATH "$ENV{HOME}/riscv" CACHE PATH "RISC-V tool path")
 
 set(RISCV_TOOLCHAIN_ROOT "${RISCV_TOOL_PATH}/toolchain/clang/${RISCV_HOST_TAG}/RISCV" CACHE PATH "RISC-V compiler path")
+set(RISCV_TOOLCHAIN_PREFIX "riscv64-unknown-linux-gnu-" CACHE STRING "RISC-V compiler prefix")
 set(CMAKE_FIND_ROOT_PATH ${RISCV_TOOLCHAIN_ROOT})
 list(APPEND CMAKE_PREFIX_PATH "${RISCV_TOOLCHAIN_ROOT}")
 
-set(CMAKE_C_COMPILER "${RISCV_TOOLCHAIN_ROOT}/bin/clang")
-set(CMAKE_CXX_COMPILER "${RISCV_TOOLCHAIN_ROOT}/bin/clang++")
-set(CMAKE_AR "${RISCV_TOOLCHAIN_ROOT}/bin/llvm-ar")
-set(CMAKE_RANLIB "${RISCV_TOOLCHAIN_ROOT}/bin/llvm-ranlib")
-set(CMAKE_STRIP "${RISCV_TOOLCHAIN_ROOT}/bin/llvm-strip")
+set(CMAKE_C_COMPILER "${RISCV_TOOLCHAIN_ROOT}/bin/${RISCV_TOOLCHAIN_PREFIX}clang")
```

Comment:
This is still not correct. The existing toolchains do not have the prefix for all the binaries.

You can test out the current toolchain/build  behavior with the docker command found in https://github.com/openxla/iree/blob/ffeaa264397dbe32127d83efce1fcf5a5930e6d4/.github/workflows/ci.yml#L829-L846

Reply:
I see there is no prefix for the clang* and binutil in docker. I'll remove these change. Thanks for the information.